### PR TITLE
Enable IGT test on Android

### PR DESCRIPTION
**Pull request by renchenglei**
Modify Android.mk file to build in Android environme
Disable GLIB & PROCS library in configuration file

### CODE REVIEW

**Comment by renchenglei** [comment]
@kalyankondapally @jennycao
PTAL.

File: lib/Android.mk [hunk 1]
```diff
@@ -33,16 +23,13 @@ LOCAL_SHARED_LIBRARIES := libpciaccess  \
...
-    LOCAL_C_INCLUDES += $(ANDROID_BUILD_TOP)/external/cairo-1.12.16/src
+    LOCAL_C_INCLUDES += external/cairo/src
     LOCAL_CFLAGS += -DANDROID_HAS_CAIRO=1 -DIGT_DATADIR=\".\" -DIGT_SRCDIR=\".\"
 else
-skip_lib_list := \
-    igt_kms.c \
-    igt_kms.h \
-    igt_fb.c
-    -DANDROID_HAS_CAIRO=0
+skip_lib_list += igt_fb.c
```
Comment (jennycao) [inline lib/Android.mk]: do not need to update this section

**Comment by kalyankondapally** [comment]
I will let @jennycao  handle the review.

**Comment by renchenglei** [comment]
Will update a new PR

**Pull request closed without merging**